ansible/roles/servers/vars/main.yml

```diff
@@ -1,9 +1,4 @@
-#role variables
-
-grafana_port: "3030"
-grafana_base_url: "http://{{ inventory_hostname }}:{{ grafana_port }}"
-grafana_username: "admin"
-grafana_password: "admin"
+# role variables
 
 # The netplan role will do nothing unless netplan_enabled is true.
 netplan_enabled: true
```

Comment:
Is it worth explaining the purpose of enabling the netplan role?

Reply:
Can do. Where should this be described?

Reply:
Also, to the date_time parameter comment - I wish it could be. The date_time appears to be an artifact of the ASM Log Publisher's "Splunk Format". Setting the LTM Request log profile to match this value is just an attempt to use the convention that ASM uses, reducing the number of distinct time dimensions indexes that ElasticSearch and Grafana have to keep track of.

Reply:
I think you could add the netplan detail to the comment above the enable flag